tags/TagMap.md

```diff
@@ -23,23 +23,69 @@ and group stats, annotate traces and logs.
 `TagValue` is a string. It MUST contain only printable ASCII (codes between
 32 and 126)
 
-## TagScope
-
-`TagScope` is used to determine the scope of a `Tag`. The values for the `TagScope` are
-Local or Request. In future, additional values can be added to address specific situations.
-
-The tag creator determines the scope of the tag.
-
-**Local Scope**
-Tag with `Local` scope are used within the process it created. Such tags are not propagated
-across process boundaries. Even if the process is reentrant the tag MUST be excluded from
-propagation when the call leaves the process.
-
-**Request Scope**
-If a tag is created with the `Request` scope then it is propagated across process boundaries subject 
-to outgoing and incoming (on remote side) filter criteria. See `TagPropagationFilter` in 
-[Tag Propagation](#Tag Propagation). Typically 'Request' tags represents a request, processing
- of which may span multiple entities.
+## TagMetadata
+
+`TagMetadata` contains properties associated with a `Tag`. For now only the property `TagTTL`
```

Comment:
Should we call it simply `TTL`? it is a key of **Tag**Metadata already.

Reply:
I think this is very language specific and implementation specific: if you make it a inner class of the TagMetadata then your proposal makes a lot of sense, or in Go where you use objects with namespace like tag.TTL.

Reply:
Perhaps this can be mentioned as a note. On a wire I'd expect it will be passed as simply `TTL` https://github.com/w3c/correlation-context/issues/4

Reply:
I'll make a note of it.